src/model/blob.rs

```diff
@@ -0,0 +1,15 @@
+use reflection::Reflection;
+use reflection_derive::Reflection;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Reflection)]
+pub struct Blob {
+    pub uri: String,
+    // TODO: change Giant API to pluralise this since it clearly returns an array
+    pub ingestion: Vec<String>,
```

Comment:
If you want to we can use `serde`'s `rename` attribute so that it's correct in the Rust but serializes to what Giant expects.

https://serde.rs/field-attrs.html

Reply:
nice